### PR TITLE
ROU-4410: Add public method VideoJumpToTime

### PR DESCRIPTION
**Pull request by bmarcelino-fe**
This PR is for adding the new feature to Video to jump to a specific time of a video.

### What was done
- A new client action called VideoJumpToTime was added to give the ability to jump to a specific time of a given video.
image

### Test Steps
**Test Case 1**
1. Go to Sample Page
2. Click on Jump To Time Video Button
3. Expected: The video will change the seconds to 6 and the Infor box will have "Code = 200 ; Message = Success; VideoJumpToTime value is equal to VideoCurrentTime: True; CurrentTime: 6"

**Test Case 2**
1. Go to Sample Page
2. Click on Jump To Time Video - Error Button
3. Expected: The video action error will be triggered

### Checklist
-   [X] tested locally
-   [X] documented the code
-   [X] clean all warnings and errors of eslint
-   [ ] requires changes in OutSystems (if so, provide a module with changes)
-   [ ] requires new sample page in OutSystems (if so, provide a module with changes)

### CODE REVIEW

**Review by rugoncalves — APPROVED** [review]
Shouldn't we also have a jumpTime? meaning 2s forward or backward.

File: src/scripts/OSFramework/OSUI/Pattern/Video/Video.ts [hunk 1]
```diff
@@ -405,6 +405,16 @@ namespace OSFramework.OSUI.Patterns.Video {
 			}
 		}
 
+		/**
+		 * uMethod to set current time
```
Comment (BenOsodrac) [inline src/scripts/OSFramework/OSUI/Pattern/Video/Video.ts:409]: uMethod typo

File: src/scripts/OutSystems/OSUI/Patterns/VideoAPI.ts [hunk 2]
```diff
@@ -189,4 +189,17 @@ namespace OutSystems.OSUI.Patterns.VideoAPI {
 
 		return result;
 	}
+
+	export function JumpToTime(videoId: string, time: number): string {
```
Comment (BenOsodrac) [inline src/scripts/OutSystems/OSUI/Patterns/VideoAPI.ts:193]: missing comments

**Review by BenOsodrac — CHANGES_REQUESTED** [review]

File: src/scripts/OutSystems/OSUI/Patterns/VideoAPI.ts (hunk 2 above)
Comment (bmarcelino-fe) [reply src/scripts/OutSystems/OSUI/Patterns/VideoAPI.ts:193]: Fixed

File: src/scripts/OSFramework/OSUI/Pattern/Video/Video.ts (hunk 1 above)
Comment (bmarcelino-fe) [reply src/scripts/OSFramework/OSUI/Pattern/Video/Video.ts:409]: Fixed

**Review by BenOsodrac — APPROVED** [review]

File: src/scripts/OutSystems/OSUI/Patterns/VideoAPI.ts [hunk 3]
```diff
@@ -189,4 +189,25 @@ namespace OutSystems.OSUI.Patterns.VideoAPI {
...
+	 * @param {number} time
+	 * @return {*}  {string}
+	 */
+	export function JumpToTime(videoId: string, time: number): string {
+		const result = OutSystems.OSUI.Utils.CreateApiResponse({
+			errorCode: ErrorCodes.Video.FailSetTime,
+			callback: () => {
+				const video = GetVideoById(videoId);
+
+				return video.setVideoJumpToTime(time);
```
[the remark this answers is not in the document]
Comment (bmarcelino-fe) [reply src/scripts/OutSystems/OSUI/Patterns/VideoAPI.ts:207]: Good catch. SOlved

**Review by OS-giulianasilva — APPROVED** [review]

**Review by BenOsodrac — APPROVED** [review]

**Pull request merged**